cmd/aws-janitor/main.go

```diff
@@ -42,7 +42,7 @@ import (
 var (
 	maxTTL                     = flag.Duration("ttl", 24*time.Hour, "Maximum time before attempting to delete a resource. Set to 0s to nuke all non-default resources.")
 	region                     = flag.String("region", "", "The region to clean (otherwise defaults to all regions)")
-	path                       = flag.String("path", "", "S3 path for mark data (required when -all=false)")
+	path                       = flag.String("path", "", "S3 path for mark data, the bucket must be marked with excludeTags (required when -all=false)")
```

Comment:
Feels a bit misleading - it does not have to be marked if bucket clean is off. Add clarification for "if s3 bucket clean is on"?